<launch>
    <node pkg="topic_tools" name="cam_throttle" type="throttle"
        args="messages main_camera/image_raw 9.0 main_camera/image_raw_throttled"/>

    <node pkg="l22_aero_vision" type="color_r_c.py" name="l22_aero_vision_node" output="screen" clear_params="true">
        <remap from="image_raw" to="main_camera/image_raw_throttled"/>
        <remap from="camera_info" to="main_camera/camera_info"/>
    </node>
    <node pkg="l22_aero_vision" type="viz.py" name="l22_aero_vision_viz_node" output="screen" clear_params="true"/>
</launch>